### DISCUSSION TITLE
No longer allowed to modify my tables

### DISCUSSION THREAD

**Discussion by joakimgk** [supabase/supabase#12730] [Questions] [answered]
Hi!

I have a small project, which I work on from time to time. After logging in now (via GitHub), I find I'm not allowed to change policies etc on tables I've previously created:

Error updating policy: failed to update pg.policies with the given ID: must be owner of table shows

What's going on here? I only have a single Supabase user (as far as I know) and I am logged in; how can I no longer be the owner of my tables? Furthermore, when I try to use the "Forgot password" (to try email login), no email is ever received.

**Comment by GaryAustin1** [comment, marked as answer]
You might be caught up in this:  https://github.com/supabase/supabase/discussions/9314 security fix.  

Also on forgot password, that I assume only works if you signed up with email and not Github.  Your login is not related to your dashboard user which is now likely `postgres` and it used to be `supabase_admin`.

**Reply by joakimgk** [reply]
Thanks for the info! This is probably the case, then... Last I worked with this database was in January, though, and then I was still able to create/delete RLS policies. It's still unclear to me what do I do now. Should I run some migration to change the ownership of the tables?  I am able to edit RLS on any _new_ table I create, so one option might be to just drop and re-create all tables and policies -- but I'm (obviously) reluctant to do this. Surely, there's some user-friendly solution to this?

**Reply by GaryAustin1** [reply]
Well they recommend contacting support.  This seems like a pretty normal use case that would have broken everyone, so something else happened in your case.

**Comment by joakimgk** [comment]
Thanks! I'll do that :) Have a good day
